workflows.tf

```diff
@@ -78,3 +100,47 @@ resource "google_workflows_workflow" "project_setup" {
   ]
 
 }
+
+# execute workflows after all resources are created
+# # get a token to execute the workflows
+data "google_client_config" "current" {
+}
+
+# # execute the copy data workflow
+data "http" "call_workflows_copy_data" {
+  url    = "https://workflowexecutions.googleapis.com/v1/projects/${module.project-services.project_id}/locations/${var.region}/workflows/${google_workflows_workflow.copy_data.name}/executions"
+  method = "POST"
+  request_headers = {
+    Accept = "application/json"
+  Authorization = "Bearer ${data.google_client_config.current.access_token}" }
+  depends_on = [
+    google_workflows_workflow.copy_data,
+    google_storage_bucket.textocr_images_bucket,
+    google_storage_bucket.ga4_images_bucket,
+    google_storage_bucket.tables_bucket
+  ]
+}
+
+# # execute the other project setup workflow
+data "http" "call_workflows_project_setup" {
+  url    = "https://workflowexecutions.googleapis.com/v1/projects/${module.project-services.project_id}/locations/${var.region}/workflows/${google_workflows_workflow.project_setup.name}/executions"
+  method = "POST"
+  request_headers = {
+    Accept = "application/json"
+  Authorization = "Bearer ${data.google_client_config.current.access_token}" }
+  depends_on = [
+    google_workflows_workflow.project_setup,
+    google_dataplex_asset.gcp_primary_textocr,
+    google_dataplex_asset.gcp_primary_ga4_obfuscated_sample_ecommerce,
+    google_dataplex_asset.gcp_primary_tables
+  ]
+}
+
+# # wait for completion
+resource "time_sleep" "wait_after_all_workflows" {
```

Comment:
do we need this resource if we are waiting on the other waiter for completion? want to make sure we dont doublecount

Reply:
Removed.